src/fs_store.rs

```diff
@@ -129,7 +129,7 @@ impl FsStore {
                 .map_err(|e| Error(ErrorKind::ErrReadingLevel1Path(entry.path(), e)))?
                 .is_file()
             {
-                tracing::warn!(
+                tracing::trace!(
```

Comment:
I believe these log levels need to be revisited. Some of them don't feel quite right...